### PR TITLE
[SageAdapter] Pass threads param as  (file) batch_size

### LINKED ISSUE #7031: SageAdapter does not pass threads

### PR DESCRIPTION
**Pull request by timosachsenberg**
## Description

forward threads parameter to "batch_size" in sage

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/topp/SageAdapter.cpp [hunk 1]
```diff
@@ -67,6 +67,8 @@ using namespace std;
 
     @brief Identifies peptides in MS/MS spectra via sage.
 
+    Setting the "threads" parameter to a non-zero value indicates how many files to load and search in parallel. Setting it to 0 defaults tp # of CPUs/2 (default in sage).
```
Comment (cbielow) [inline src/topp/SageAdapter.cpp]:
```suggestion
    Setting the "threads" parameter to a non-zero value indicates how many files to load and search in parallel. Setting it to 0 defaults to # of CPUs/2 (default in sage).
```

**Comment by cbielow** [comment]
can we please make PR titles more descriptive?! This help a lot when reviewing PR's for a release to make sure everything is in the release notes.
Thanks!

**Comment by timosachsenberg** [comment]
sorry and thanks for the rename of the issue... just trying to squeeze in a few improvements during my vacation ;) .

**Comment by jpfeuffer** [comment]
Needs rebase and according to the author this needs to be handled by an env var to the process instead.
Or wait for a threads param.

**Comment by ypriverol** [comment]
@timosachsenberg @jpfeuffer is this PR now surpassed by #7577 ?

**Comment by jpfeuffer** [comment]
Not sure. But I am also not sure if the threads parameter is the best equivalent for batch_size. It's not really the same.

**Comment by timosachsenberg** [comment]
agree that it is not the same. should be handled through env

**Pull request closed without merging**